api/client/stats_helpers.go

```diff
@@ -150,7 +154,7 @@ func (s *containerStats) Collect(cli client.APIClient, streamStats bool, waitFir
 				s.mu.Lock()
 				s.err = err
 				s.mu.Unlock()
-				return
+				continue
```

Comment:
I can change this behavior if we really want to, but is beyond the intention of the PR.